build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 ext{
-    commercetoolsJvmSdkVersion = '1.59.0'
+    commercetoolsJvmSdkVersion = '1.60.0'
```

Comment:
Could you please update README docs, check for `1.57.0`.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/676/commits/973751563e0ce3f34c79a1c8fd3a675d16c76ce9